test/iron-location.html

```diff
@@ -164,6 +167,76 @@
         assertHaveSameUrls(urlElem, otherUrlElem);
       });
     });
+    suite('when intercepting links', function() {
+
+      /**
+       * Returns whether an iron-location element with the given urlSpaceRegex
+       * if any would prevent the default behavior when the given anchor element
+       * is clicked.
+       */
+      function clickWouldBeCaptured(anchor, urlSpaceRegex) {
```

Comment:
The tense in the phrasing makes the intended usage a little ambiguous to me. I'm not sure if result reflects..

 a. What the iron-location should do in the ideal case for this anchor
 b. What the iron-location will actually do as implemented for this anchor

On the surface, I hoped it was b, and in reading the implementation, I learned that it is b. I think using the word `would` which sounds speculative is just mildly ambiguous to me compared to something more affirmative like `does`.

Anyway, that's a lot of words for a minor piece of feedback. No actual action required, but please consider the feedback!